examples/apps/echo/README.md

```diff
@@ -0,0 +1,49 @@
+# Echo Demo Application
+
+This simple demo application echos back the same message sent from
+the client to the worker.
+
+## Using the Echo Command Line Client
+
+The command line client, `echo_client.py` sends a message to the worker,
+which echos back the same message back to the client.
+For command line options, type `./echo_client.py -h` from the
+`client` directory.
+
+To use:
+
+1.  If needed, update the Ethereum account and direct registry contract
+    information in `docker/Dockerfile.tcf-dev` and
+    `examples/common/python/connectors/tcf_connector.toml`
+2.  Follow instructions in the "Docker-based Build and Execution" section of
+    the [build document](../../../BUILD.md#dockerbuild) through step 4
+    (activating a virtual environment)
+3.  Terminal 1 is running the TCF Enclave Manager and Listener with
+    `docker-compose` . Terminal 2 is running the Docker container shell
+    with the `(_dev)` prompt
+4.  In Terminal 2 run `cd $TCF_HOME/examples/apps/echo/client`
+5.  In Terminal 2 install the Solidity compiler as follows:
+    ```bash
+    mkdir -p $HOME/.py-solc/solc-v0.4.25/bin \
+    && curl -LsS https://github.com/ethereum/solidity/releases/download/v0.4.25/solc-static-linux \
+            -o /root/.py-solc/solc-v0.4.25/bin/solc \
+    && chmod 0755 /root/.py-solc/solc-v0.4.25/bin/solc &&
+    export SOLC_BINARY=/root/.py-solc/solc-v0.4.25/bin/solc
```

Comment:
Can we add these env variables to /etc/environment so that these variables will be auto set for every docker shells?

Reply:
The problem is the solc compiler is not installed for the docker bash shell (see above).

If that problem is fixed, how do I add it?  Do I edit `/etc/environment` somewhere?
Or do I just add a line in `docker-compose.yaml` under the `environment:` stanza?

Reply:
I am able to run echo client with Ethereum with below option
_dev) root@0840863d8f74:/project/TrustedComputeFramework/examples/apps/echo/client# python echo_client.py --config ../../../common/python/connectors/tcf_connector.toml